ProteoformSuiteInternal/ProteoformCommunity.cs

```diff
@@ -96,32 +141,145 @@ private static List<ProteoformRelation> count_nearby_relations(List<ProteoformRe
         public Dictionary<string, List<ProteoformRelation>> relate_ed()
         {
             Dictionary<string, List<ProteoformRelation>> ed_relations = new Dictionary<string, List<ProteoformRelation>>();
-            Parallel.ForEach(decoy_proteoforms, decoys =>
+            if (!Lollipop.limit_TD_BU_theoreticals)
             {
-                ed_relations[decoys.Key] = relate_et(experimental_proteoforms, decoys.Value, ProteoformComparison.ed);
-            });
+                Parallel.ForEach(decoy_proteoforms, decoys =>
+                {
+                    ed_relations[decoys.Key] = relate_et(experimental_proteoforms.Where(p => p.accepted).ToArray(), decoys.Value, ProteoformComparison.ed);
+                });
+            }
+            else
+            {
+                Parallel.ForEach(decoy_proteoforms, decoys =>
+                {
+                    ed_relations[decoys.Key] = relate_et(experimental_proteoforms.Where(p => p.accepted).ToArray(), decoys.Value.Where(t => t.psm_count_BU > 0).ToArray(), ProteoformComparison.ed);
+                });
+            }
             return ed_relations;
         }
-        public List<ProteoformRelation> relate_unequal_ee_lysine_counts()
+
+        public List<ProteoformRelation> relate_ef()
         {
             List<ProteoformRelation> ef_relations = new List<ProteoformRelation>();
-            ExperimentalProteoform[] pfs1 = new List<ExperimentalProteoform>(this.experimental_proteoforms).ToArray();
-            ExperimentalProteoform[] pfs2 = new List<ExperimentalProteoform>(this.experimental_proteoforms).ToArray();
+            ExperimentalProteoform[] pfs1 = new List<ExperimentalProteoform>(this.experimental_proteoforms.Where(p => p.accepted)).ToArray();
```

Comment:
Where do we accept/reject experimentals again? It seems like this makes sense, but why only use accepted ones (for record's sake)?